manifests/gcp_marketplace/guide.md

```diff
@@ -0,0 +1,65 @@
+# Deploy Kubeflow Pipelines from Google Cloud Marketplace
+
+You can go to [Google Cloud Marketplace](https://console.cloud.google.com/marketplace) to deploy Kubeflow Pipelines via UI.
```

Comment:
Add: To get to Kubeflow Pipelines page in Marketplace, enter "Kubeflow Pipelines" in the search box in Marketplace landing page. Alternatively you can access the [Marketplace page for Kubeflow Pipelines](https://pantheon.corp.google.com/marketplace/details/google-cloud-ai-platform/kubeflow-pipelines) directly.